### ISSUE TITLE
Cancel out drag action programmatically?

### ISSUE THREAD

**Issue by irrg** [Vivify-Ideas/vue-draggable#26]
Hey @nikolasp, sorry to bug you again 😀

Making progress with integrating the current release but will look at your 2.0.0 beta on Monday. I was wondering if there's an obvious way that I'm not seeing to 'undo' a drag programmatically? 

I know that the reactive data isn't changed yet until I do so myself, but I'm trying to have a 'confirm'/'undo' step in the drag and my first idea, forcing a redraw, didn't seem to do anything.

**Comment by nikolasp** [comment]
Hi @irrg,

No problem at all. 😄 
I'm not sure what exactly behaviour you want to achieve, but let's suppose that you just want to prevent state changing. You can use callbacks `onDragstart` or `onDragend` and basically call stop method from event. 

I've updated last example with `onDragend` callback to prevent D&D. Please take a look at example...
[![Edit VueDraggable Reactivity](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/8nkk8pw6x0)

**Comment by irrg** [comment]
I'll try to be more clear, I was pretty tired when I wrote that 😜

I'm going to have multiple ULs that have multiple LIs. I click a button inside the LI to make it `modifiable` which is my target class name for the options. It then shows a handle, a 'confirm' and a 'cancel' button.

The user can drag the `li.modifable` into another UL but after finishing the drag, they can click the 'cancel' inside the LI to undo the move. So, the event finishes, but I'm not sure if there's an obvious way I could "undo" the drag action after it finished or if it's best to track the state myself and somehow move it back to where it was programmatically.

**Comment by nikolasp** [comment]
Hi @irrg,

This behaviour is out of the scope for this lib, but I've updated example to your use case. It's not totally by your description, but I think that I've covered the most important part. Basically you want possibility to return to the previous state. You just need to persist previous state and when user clicks on a cancel button you will basically replace the current state with old one and it will automatically redraw lists.
Take a look at modified example [![Edit VueDraggable Reactivity](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/8nkk8pw6x0)
